help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -92,6 +92,11 @@ To configure details for the destination, fill in the required and optional fiel
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
 *  **[!UICONTROL Name of your source partner]**: Your customer/company name. Only supported Magnite Streaming clients are available for selection.
 
+>[!NOTE]
```

Comment:
Please rephrase this note as such:

`The source partner name must be an all-lowercase string which matches the name of the Amazon S3 delivery bucket you have configured with Magnite and set up in the [authenticate to destination](#authenticate) step. The supported characters include 'a-z', '0-9', '-'(dash), or '_'(underscore).`

Reply:
Changed the Note due to the new requirement that allows uppercase letters (that later get converted to lowercase). Please review. Thanks!